packages/components/src/internal/components/ontology/OntologyBrowserPanel.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+
+import { LoadingSpinner } from '../base/LoadingSpinner';
+
+import { OntologyTabs } from './OntologyTabs';
+import { ConceptInformationTabs, ConceptInfoTabs } from './ConceptInformationTabs';
+import { getOntologyDetails } from './actions';
+import { ConceptModel, OntologyModel } from './models';
+
+export interface OntologyBrowserProps {
+    ontologyId?: number;
+}
+
+interface ConceptPathModel {
+    path: string;
+    conceptCode: string;
+    children: ConceptPathModel[];
+}
+
+// TODO remove when this becomes dynamic
+const mockConcept = {
+    name: 'widgets',
+    code: 'CodedWidgets',
+    paths: undefined,
+    description: 'I am a big long description',
+} as ConceptModel;
+
+export const OntologyBrowserPanel: FC<OntologyBrowserProps> = props => {
+    const { ontologyId } = props;
+
+    const [ontology, setOntologyModel] = useState<OntologyModel>();
+    const [conceptTab, setActiveTab] = useState<ConceptInfoTabs>();
+    const [selectedConcept, setSelectedConcept] = useState<ConceptModel>();
+
+    useEffect(() => {
+        getOntologyDetails(ontologyId).then(details => {
+            setOntologyModel(details);
+            // setOntologyName(details.name);
+            // setConceptCount(details.conceptCount);
+        });
+    }, [setOntologyModel]);
+
+    useEffect(() => {
+        setSelectedConcept(mockConcept);
+    }, [setSelectedConcept]);
+
+    if (!ontology) {
+        return <LoadingSpinner />;
+    }
+
+    return (
+        <>
+            <div className="panel panel-default ontology-browser-container">
```

Comment:
Since we will have a case where this component is rendered in a dialog (in which case the ontology title goes in the dialog header) vs rendering this component on the page (in which case I think we'll want to have the ontology title in a panel header), I would likely do something along these lines to add the panel-heading for now:

```
            <div className="panel panel-default ontology-browser-container">
                <div className="panel-heading">
                    {ontology.name}
                </div>
                <div className="panel-body">
                ...
                </div>
            </div>
```